packages/ni.measurementlink.measurement.v1.proto/pyproject.toml

```diff
@@ -32,7 +32,7 @@ packages = [{include = "ni", from = "src"}]
 [tool.poetry.dependencies]
 python = "^3.9"
 protobuf = {version=">=4.21"}
-nitypes = {version=">=0.1.0dev3", allow-prereleases=true}
+ni-measurementlink-proto = ">=0.1.0dev0"
```

Comment:
Since this lives in the same monorepo, you probably ought to add a 2nd definition to dev.dependencies with path="../../whatever"